<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Associar Produtos a Campo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Campo:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="campoCb"/>
   </item>
   <item>
    <widget class="QGroupBox" name="produtosGroup">
     <property name="title">
      <string>Selecionar Produtos</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Lote:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="loteCb"/>
        </item>
        <item>
         <widget class="QPushButton" name="carregarProdutosBtn">
          <property name="text">
           <string>Carregar Produtos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="filtroLe">
          <property name="placeholderText">
           <string>Filtrar produtos (nome, MI, INOM...)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filtrarBtn">
          <property name="text">
           <string>Filtrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="produtosTable">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>MI</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>INOM</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Escala</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dump</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="infoLabel">
          <property name="text">
           <string>Selecione os produtos para associar ao campo.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="quantidadeLabel">
          <property name="text">
           <string>0 produto(s) selecionado(s)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Associar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>